metashare/api/models.py

```diff
@@ -289,6 +294,29 @@ def save(self, *args, **kwargs):
 
         return ret
 
+    def refresh_access_token(self):
+        """
+        This refreshes the access token in-band, so be careful.
+        """
+        self.config = refresh_access_token(
+            config=self.config, org_name="dev", login_url=self.login_url
+        )
+        self.save()
+
+    def get_login_url(self):
+        self.refresh_access_token()
+        org_config = OrgConfig(self.config, "dev")
+        session = jwt_session(
+            settings.SF_CLIENT_ID,
+            settings.SF_CLIENT_KEY,
+            org_config.username,
+            url=self.login_url,
+        )
```

Comment:
This raises an error:

```
Bad Request for url: https://cs92.salesforce.com/services/oauth2/token: {"error":"invalid_grant","error_description":"audience is invalid"}
```

Reply:
Since it's a scratch org, you need to pass is_sandbox=True so that it will use test.salesforce.com instead of login.salesforce.com as the `aud`